azurerm_app_service_certificate - introduce argument app_service_plan_id for usage with ASE (#12608)
hashicorp/terraform-provider-azurerm: website/docs/r/app_service_certificate.html.markdown

* `password` - (Optional) The password to access the certificate's private key. Changing this forces a new resource to be created.

> * `app_service_plan_id` - (Optional) Resource ID of the associated App Service plan. Must be specified when the certificate is used inside an App Service Environment hosted App Service. Changing this forces a new resource to be created.

Reviewer: ```suggestion
* `app_service_plan_id` - (Optional) The ID of the associated App Service plan. Must be specified when the certificate is used inside an App Service Environment hosted App Service. Changing this forces a new resource to be created.
```

* `thumbprint` - The thumbprint for the certificate.

> * `hosting_environment_profile_id` - The ID of the the App Service Environment to use for the certificate.

Reviewer: ```suggestion
* `hosting_environment_profile_id` - The ID of the the App Service Environment where the certificate is in use.
```
